### PR TITLE
cleanUp on "stable" webpage

### PR DESCRIPTION
**Pull request by i-walker**
fixes some webpage problems users have and deletes references to SnapShot builds.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,7 +1,6 @@
 <img height="100" src="https://avatars2.githubusercontent.com/u/29458023?v=4&amp;s=200" width="100">
 
-[![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.arrow-kt/arrow-core/badge.svg)](https://maven-badges.herokuapp.com/maven-central/io.arrow-kt/arrow-core)
-[![Build Status](https://travis-ci.org/arrow-kt/arrow.svg?branch=master)](https://travis-ci.org/arrow-kt/arrow/)
+[![Maven Central](https://img.shields.io/maven-central/v/io.arrow-kt/arrow-docs.svg?label=Maven%20Central)](https://search.maven.org/search?q=g:%22io.arrow-kt%22%20AND%20a:%22arrow-docs%22)
```
Comment (i-walker) [inline README.md]: This is one mavenCentral Badge with a 0.9.0 label

File: modules/docs/arrow-docs/docs/docs/README.md [hunk 2]
```diff
@@ -90,7 +56,7 @@ dependencies {
     compile "io.arrow-kt:arrow-extras-data:$arrow_version"
     compile "io.arrow-kt:arrow-extras-extensions:$arrow_version"
     kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-
+
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/README.md]: Unnecessary whitespace

**Comment by pakoito** [comment]
Why aren't reviews necessary to merge here?

File: README.md (hunk 1 above)
Comment (pakoito) [reply README.md]: The badge updates with the version in central. Is it necessary?

**Comment by i-walker** [comment]
We're still working on a way to block it, but that's just for now..

File: README.md (hunk 1 above)
Comment (i-walker) [reply README.md]: Yes, it is as of today it only displays the 0.8.2 maven central,  which is odd considering that 0.9.0 is the current stable version.

**Comment by raulraja** [comment]
That is because the referenced artifact is wrong. We discussed before that
arrow-core does not exist in 0.9.0. this badge should point to
arrow-core-data currently and back to arrow-core in 0.10.0

**Comment by i-walker** [comment]
Does this branch need any changes or do you approve?

File: modules/docs/arrow-docs/docs/docs/README.md [hunk 3]
```diff
@@ -111,6 +76,7 @@ dependencies {
     compile "io.arrow-kt:arrow-query-language:$arrow_version" //optional
     compile "io.arrow-kt:arrow-integration-retrofit-adapter:$arrow_version" //optional
 }
+
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/README.md]: Additional line

File: modules/docs/arrow-docs/docs/docs/README.md [hunk 4]
```diff
@@ -90,7 +56,6 @@ dependencies {
     compile "io.arrow-kt:arrow-extras-data:$arrow_version"
     compile "io.arrow-kt:arrow-extras-extensions:$arrow_version"
     kapt    "io.arrow-kt:arrow-meta:$arrow_version"
-
```
Comment (pakoito) [inline modules/docs/arrow-docs/docs/docs/README.md]: Missing line between mandatory and optional
